src/utilities/__tests__/findSchemaChanges-test.ts

```diff
@@ -43,6 +60,30 @@ describe('findSchemaChanges', () => {
     ]);
   });
 
+  it('should detect a field changing description', () => {
+    const oldSchema = buildSchema(`
+      type Query {
+        foo: String
+        bar: String
+      }
+    `);
+
+    const newSchema = buildSchema(`
+      type Query {
+        foo: String
+        "New Description"
+        bar: String
+      }
+    `);
+    expect(findSchemaChanges(oldSchema, newSchema)).to.deep.equal([
+      {
+        type: SafeChangeType.DESCRIPTION_CHANGED,
+        description:
+          'Description of field Query.bar has changed to "New Description".',
```

Comment:
Generally -- not just here -- we could consider printing the old description as well, giving better context as to what exactly changed.